### PR TITLE
Fixes oscap long-run test for 6.2.z, fixes issue #4286

### PR DESCRIPTION
**Pull request by san7ket**
Fixed locators , and other nitpicks test is able to run again.

```
 nosetests tests.foreman.longrun.test_oscap:OpenScapTestCase.test_positive_upload_to_satellite
.
----------------------------------------------------------------------
Ran 1 test in 1045.552s

OK

```

Signed-off-by: san7ket <san7ketjagtap@gmail.com>

### CODE REVIEW

File: tests/foreman/longrun/test_oscap.py [hunk 1]
```diff
@@ -26,6 +27,7 @@
 from robottelo.constants import (
     ANY_CONTEXT,
     DEFAULT_SUBSCRIPTION_NAME,
+    DEFAULT_LOC,
```
Comment (sghai) [inline tests/foreman/longrun/test_oscap.py]: alphabetical order ?

**Comment by sghai** [comment]
ACK.. non blocking comment

**Comment by rplevka** [comment]
ACK

**Comment by oshtaier** [comment]
Good job
ACK

**Comment by san7ket** [comment]
@sghai addressed the comments

**Pull request merged**